Redis/node.deleteKey.js

```diff
@@ -0,0 +1,17 @@
+let logger = global.logger;
+
+module.exports = async (connectorData, inputData) => {
+    logger.trace("Redis Delete Value Node: Invoked!");
```

Comment:
Please change the log lines to "Delete key". You are not deleting the value. But the key.